tests_integration/__tests__/file-info.js

```diff
@@ -106,16 +106,15 @@ describe("extracts file-info with inferredParser=foo when a plugin is hand-picke
 });
 
 test("API getFileInfo with no args", () => {
-  // TODO: change this to `rejects.toThrow()` when we upgrade to Jest >= 22
-  // https://github.com/facebook/jest/issues/3601
-  expect.assertions(1);
-  return prettier.getFileInfo().catch(err => {
-    expect(err).toBeDefined();
-  });
+  expect(prettier.getFileInfo()).rejects.toThrow(
```

Comment:
you need to `return` or `await` it - it's async

Reply:
you are right, I forgot, fixed